### DISCUSSION TITLE
عمومی

### DISCUSSION THREAD

**Discussion by hayyaun** [hayyaun/kids#1] [General]
### سلام به همگی 👋

داخل این گروه میتونید راجبه هر موضوعی صحبت کنید 

#### سوال و جواب

لطفا هر ایرادی دارید اینجا بفرستید
تا من یا بقیه دوستاتون ببینیم و رفع کنیم
هر سوال یا جواب مفید یک کارما ⭐ داره

**Comment by Nikannejatifar** [comment]
سلام اگه میشه بعد از هر تغییر بگید که ما فایل زیپ رو دانلود کنیم

**Reply by hayyaun** [reply]
چشم نیکان جان به نکته درستی اشاره کردی حتما این کار رو میکنم.
آپدیت: به بالای دکمه دانلود اگه دقت کنید سبز نوشته که چند روز قبل تغییرات داشتیم

**Comment by Nikannejatifar** [comment]
میشه برای اچیومنت ها توضیح بزارید و بگید که چجوری میشه اونا رو گرفت؟

**Reply by hayyaun** [reply]
باشه حتما تو جدولی که داخل سایت هست ازین به بعد اگه موستون روی هر آیکونی ببرید بهتون نشون میده که اسم اچیومنت چیه و برای چی بهتون داده شده.

**Comment by sniper1390** [comment]
سلام. ممنون که من را دعوت کردین.

**Reply by hayyaun** [reply]
خوب کاری کردم!

**Comment by Noyangolalizade** [comment]
سلام استاد اگه میشه یکاری کنید بشه تو سایت هم پیام داد و فقط تو گیت هاب نباشه

**Reply by hayyaun** [reply]
سلام نویان جان، اگه این کارو کنم ربات ها میتونن فلاد کنن سایت رو و از نظر امنیتی ایراد داره. یا باید حساب کاربری تو سایت خودم بذارم ایجاد کنید و نمیخام اینکارو کنم چون سایت شخصی هست نه تالار گفتگو.

**Comment by Noyangolalizade** [comment]
سلام،اگه میشه کسایی که به بازی پینگ پونگ دعوت شدن رو از جدول حذف کنید که برای بقیه امید بشه،اون جوری استرس میگیرن فک میکنن نمیتونن

**Reply by hayyaun** [reply]
آره ایده خوبیه. منتقلتون میکنم به انتهای جدول تا اطلاع ثانوی. چون نمیخام لیست اچیومنتاتون از بین بره.
اما اگه هر چهارتاتون بخاید که حذفتون کنم اینکارو میکنم.
